django/db/models/base.py

```diff
@@ -40,6 +37,16 @@
 from django.utils.version import get_version
 
 
+class Deferred(object):
+    def __repr__(self):
+        return '<Deferred field>'
+
+    def __str__(self):
+        return '<Deferred field>'
```

Comment:
I should have used `@python_2_unicode_compatible` in the first place. I can't think of any issues since there aren't any non-ASCII chars in these strings.